shardingsphere-control-panel/shardingsphere-orchestration/shardingsphere-orchestration-core/shardingsphere-orchestration-core-configcenter/src/main/java/org/apache/shardingsphere/orchestration/core/configcenter/listener/SchemaChangedListener.java

```diff
@@ -109,7 +108,7 @@ private ShardingOrchestrationEvent createUpdatedEvent(final String shardingSchem
     }
     
     private ShardingOrchestrationEvent createUpdatedEventForExistedSchema(final DataChangedEvent event, final String shardingSchemaName) {
-        return event.getKey().equals(configurationNode.getDataSourcePath(shardingSchemaName)) 
+        return event.getKey().equals(configurationNode.getDataSourcePath(shardingSchemaName))
```

Comment:
it's better to try these functions work well via a real zk.

Reply:
i have alreay verify